release/setup.cfg

```diff
@@ -0,0 +1,29 @@
+[metadata]
+name = git-filter-repo
+url = https://github.com/newren/git-filter-repo
+project_urls =
+    Source = https://github.com/newren/git-filter-repo
+    Issues = https://github.com/newren/git-filter-repo/issues/
+description = git filter-branch replacement
```

Comment:
Hmm...I listed that up-front on the GitHub page because it very quickly explains to people what it does.  But it's a description with an expiration date; I expect people will eventually not be familiar with filter-branch anymore.  And I'm not sure if it makes as much sense for the PyPI audience.  Maybe we should use something more generic such as "Quickly rewrite git repository history"?